nf_core/lint/nextflow_config.py

```diff
@@ -262,6 +268,48 @@ def nextflow_config(self):
                 )
             )
 
+    # Check if custom profile params are set correctly
+    if self.nf_config.get("params.custom_config_version", "").strip("'") == "master":
+        passed.append("Config `params.custom_config_version` is set to `master`")
+    else:
+        failed.append("Config `params.custom_config_version` is not set to `master`")
+
+    custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/" + self.nf_config.get(
+        "params.custom_config_version", ""
+    ).strip("'")
```

Comment:
I generally try to avoid adding strings and use `format()` or f-strings instead, as it tends to be safer so a good habit to be in.

```suggestion
    custom_config_base = "https://raw.githubusercontent.com/nf-core/configs/{}".format(self.nf_config.get(
        "params.custom_config_version", ""
    ).strip("'"))
```